### ISSUE TITLE
Не работает звук и видео

### ISSUE THREAD

**Issue by linuxbuh** [deemru/chromium-gost#33]
Версия 91.0.4472.114
Linux
та-же версия Chromium - работает и звук и видео.

**Comment by linuxbuh** [comment]
Странно - решилось запуском оригинального Chromium. После запуска Chromium  у Chromium Gost стало работать и видео и звук.

**Comment by linuxbuh** [comment]
Видимо в папке с настройками что-то изменилось

**Comment by linuxbuh** [comment]
Нет. Если Оригинальный Chromium запущен и запустить Chromium Gost - то виде и звук работают. Если запустить только ChromiumGost - то видео и звук НЕ РАБОТАЮТ
![image](https://user-images.githubusercontent.com/34405079/122847827-0ebb6900-d311-11eb-8d8f-74f04ae99471.png)

**Comment by snejok** [comment]
Посмотрите тут #23
